chipsec/modules/common/secureboot/variables.py

```diff
@@ -73,7 +73,7 @@ def is_supported(self):
     def can_modify(self, name, guid, data, attrs):
         self.logger.log("    > Attempting to modify variable {}:{}".format(guid, name))
 
```

Comment:
Should there be a check that data length is long enough to be modified?

Reply:
I'll look into it, but for now, this works. =)